src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line at the beginning of the class definition.